tests/js/helpers/helpers-caseholder.js

```diff
@@ -0,0 +1,60 @@
+/*
+
+  Test the convenience caseholder we provide.  Wiring in sequence steps at the start of the sequence is well exercised
+  by other tests in this package, these tests focus more on steps appended to the end of test sequences and on
+  confirming that everything works end-to-end.
+
+ */
+"use strict";
+var fluid  = require("infusion");
+var jqUnit = require("node-jqunit");
```

Comment:
I removed this line which was causing a linting failure and pushed - thanks for all your hard work on this pull, ADTKINS!